### PR TITLE
Cockos.REAPER version 6.43

### PR DESCRIPTION
**Pull request by LeonarddeR**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38590)

### CODE REVIEW

File: manifests/c/Cockos/REAPER/6.43/Cockos.REAPER.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+InstallerSuccessCodes:
+- 1223
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://www.reaper.fm/files/6.x/reaper643_x64-install.exe
+  InstallerSha256: A92DF108A0CE8C10457E626C6E07D75DBA8DDF330A541AA506B67AC1B4F0AE1C
+  UpgradeBehavior: install
```
Comment (ItzLevvie) [inline manifests/c/Cockos/REAPER/6.43/Cockos.REAPER.installer.yaml:20]:
```suggestion
  UpgradeBehavior: install
  ProductCode: REAPER
```

File: manifests/c/Cockos/REAPER/6.43/Cockos.REAPER.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://www.reaper.fm/files/6.x/reaper643-install.exe
+  InstallerSha256: 9F0B869F4125B20D0A18564EA02F7E1E2731A99C99601907AF9D707DEC2A019A
+  UpgradeBehavior: install
```
Comment (ItzLevvie) [inline manifests/c/Cockos/REAPER/6.43/Cockos.REAPER.installer.yaml:27]:
```suggestion
  UpgradeBehavior: install
  ProductCode: REAPER
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by leonardder** [comment]
@ItzLevvie Coming back to this, I'm not convinced that ProductCode has any positive benefit. See https://github.com/microsoft/winget-create/issues/212#issuecomment-1011052431

**Comment by ItzLevvie** [comment]
cc @leonardder: For this package, `ProductCode` is required because the application doesn't doesn't display a publisher in Control Panel; this means that WinGet cannot recognize the application which means users are unable to upgrade the application.

This is why we need the `ProductCode` for this package so it can be listed properly in `winget list`.

**Comment by leonardder** [comment]
Thanks for pointing this out. I wonder whether it might be an option to 
contact Cockos about this, since they also fail to register the proper 
version number in the registry. I'm happy to do that.

**Pull request merged**